helpers/necesidadHelper/consecutivo.go

```diff
@@ -0,0 +1,91 @@
+package necesidadhelper
+
+import (
+	models_configuracion "github.com/udistrital/configuracion_api/models"
+	models_consecutivos "github.com/udistrital/consecutivos_crud/models"
+	consecutivohelper "github.com/udistrital/plan_cuentas_mid/helpers/consecutivoHelper"
+)
+
+func CrearConsecutivo(vigencia int) (id int, outputError map[string]interface{}) {
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "CrearConsecutivo - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	if idProceso, err := GetIdProcesoNecesidad(); err != nil {
+		outputError = err
+	} else {
+		if consecutivo, err := SolicitudConsecutivo(vigencia, idProceso); err != nil {
+			outputError = err
+		} else {
+			id = consecutivo.Consecutivo
+		}
+	}
+
+	return id, nil
```

Comment:
Revisar: En caso de fallar alguna de las lineas L21 o L24, se estaría retornando igualmente `nil`, posiblemente baste con
```suggestion
	return
```

Reply:
Cambiado.